lib/tasks/database.rake

```diff
@@ -0,0 +1,74 @@
+require "mysql2"
+require "yaml"
+require "nokogiri"
+require "reverse_markdown"
+
+namespace :database do
+  namespace :legacy do
+    # NOTE to run remotely on heroku:
+    # Source: https://devcenter.heroku.com/articles/rake
+    #
+    # $ heroku run rake database:legacy:import_news
+    #
+    desc "Import news records to database"
+    task :import_news => :environment do
+      STDOUT.puts "This will destroy your news_items table. Enter 'Y' to confirm: [y/N]"
+      input = STDIN.gets.chomp
+      raise "Aborting import. You entered #{input}" unless input.downcase == "y"
+
+      NewsItem.destroy_all
+      records = YAML.load_file "#{Rails.root}/db/legacy.yml"
+      records.each do |data|
+        news_item = NewsItem.new
+        published_at = data["post_date"]
```

Comment:
Not quite what I meant -- the key should exists, but the value might be nil :-) I should have clarified that this would be for all keys we're accessing in the hash. But happy to move on given it's a one-off task.